recipes/xtensor/all/conanfile.py

```diff
@@ -0,0 +1,88 @@
+import os
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class XtensorConan(ConanFile):
+    name = "xtensor"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xtensor-stack/xtensor"
+    description = "C++ tensors with broadcasting and lazy computing"
+    topics = ("conan", "numpy", "multidimensional-arrays", "tensors")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "xsimd": [True, False],
+        "tbb": [True, False],
+        "openmp": [True, False],
+    }
+    default_options = {"xsimd": True, "tbb": False, "openmp": False}
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = self.name + "-" + self.version
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def configure(self):
+        if self.options.tbb and self.options.openmp:
+            raise ConanInvalidConfiguration(
+                "The options 'tbb' and 'openmp' can not be used together."
+            )
+
+        # https://github.com/xtensor-stack/xtensor/blob/master/README.md
+        # - On Windows platforms, Visual C++ 2015 Update 2, or more recent
+        # - On Unix platforms, gcc 4.9 or a recent version of Clang
+        version = tools.Version(self.settings.compiler.version)
+        compiler = self.settings.compiler
+        if self.settings.compiler.cppstd and not any(
+            [
+                str(self.settings.compiler.cppstd) == std
+                for std in ["14", "17", "20", "gnu14", "gnu17", "gnu20"]
+            ]
+        ):
+            raise ConanInvalidConfiguration("xtensor requires at least C++14")
+        elif compiler == "Visual Studio":
+            if version < "16":
+                raise ConanInvalidConfiguration(
+                    "xtensor requires at least Visual Studio version 15.9, please use 16"
+                )
+            if not any(
```

Comment:
remove it, cppstd is not configured by default, it will fail on Windows for default profiles.